app/event_images/event_images.py

```diff
@@ -0,0 +1,148 @@
+from nltk.tokenize import word_tokenize
+from typing import Optional
+from word_forms.lemmatizer import lemmatize
+
+import re
+
+
+IMAGES_LINK_MAP = {
```

Comment:
I'd like to suggest an idea so you could discard this dict:
1. Rename all the images to be with exactly the same name as the key (for ex.: cycling.jpg -> cycle.jpg).
2. Create an auxiliary function which receives the key and generates the full path: f"event_flairs\{key}.jpg".

Also, note that you are using "\".
It's better to verify on which os the application is running on.
Even better - support all os platforms :)

Reply:
good idea! I'll do that!

Reply:
can you hint me how do I support all os platforms?

Reply:
Try to google it :)